build.txt

```diff
@@ -2,16 +2,23 @@
 Prerequisites
```

Comment:
Looks like this file contains duplicated content and should be dropped. 
If for some reason we can't remove it today we can put "read Readme.md" inside

Reply:
I think we can remove this file at all